includes/classes/Indexable.php

```
@@ -1210,7 +1210,7 @@ public function get_distinct_meta_field_keys( $blog_id = null ) {
 			$meta_keys = array_values( array_keys( $meta_fields ) );
 			sort( $meta_keys );
 		} catch ( \Throwable $th ) {
-			return new \Exception( 'Meta fields not available.', 0 );
+			throw new \Exception( 'Meta fields not available.', 0 );
```

@felipeelia This error wasn't being caught in the API endpoint because the API endpoint expected a thrown exception. As far as I'm aware exceptions should be thrown rather than returned, but let me know if there's a better fix.